datalad_dataset_addition_procedure.md

```diff
@@ -1,6 +1,22 @@
-# Datalad dataset addition procedure
+# Datalad dataset addition procedures
 
-This procedure is intended for developers.
+This documentation is intended for developers.
```

Comment:
I don't think that's the right term. The DataLad route is intented to anyone that can use a CLI, not necessarily developers. The Zenodo route is for everyone.

Reply:
I would just remove this comment and add a line on CLI requirements at the begining of "Upload using DataLad"/